app/helpers/alchemy/admin/contents_helper.rb

```diff
@@ -116,10 +116,9 @@ def render_locale_links_for_content(content)
           end
         end
         if links.present?
-          links.unshift('Other translations:')
+          link_string = links.join(' | ')
         end
-        links.join(' | ')
-
+        'Other translations: ' + link_string
```

Comment:
link_string could be nil.

links.blank? ? "" : "Other translations:  #{links.join(' | ')}"